src/main.ts

```diff
@@ -14,6 +14,10 @@ const root =
     ? path.join(process.env.GITHUB_WORKSPACE, ".") // Running on GH but not tests
     : path.join(__dirname, "..") // default, but should never trigger
 
+const userAgentVersion = process.env.GITHUB_ACTION_REF
```

Comment:
Would this also work?

```suggestion
const userAgentVersion = process.env.GITHUB_ACTION_REF || "unknown";
```